types/intl-locales-supported/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for intl-locales-supported 1.0
+// Project: https://github.com/yahoo/intl-locales-supported
+// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function areIntlLocalesSupported(locales: string | string[]): void;
```

Comment:
This is not a `default` export according to the [source](https://github.com/yahoo/intl-locales-supported/blob/master/index.js#L8). `module.exports = ` equates to `export =` in TypeScript.

Reply:
done